hr_family/models/hr_employee.py

```diff
@@ -19,20 +19,27 @@
 #
 #
 
-from openerp import models, fields
+from odoo import models, fields
 
 
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
-    fam_spouse = fields.Char("Name")
-    fam_spouse_employer = fields.Char("Employer")
-    fam_spouse_tel = fields.Char("Telephone.")
+    fam_spouse = fields.Char(string="Spouse's Name")
```

Comment:
@adrienpeiffer I asked @GillesTephaneMeyomesse to add the "Spouse" in the def of the fields to be more clear when you will export info into csv,xls, but it's fine for me to add labels in the view and to be shown without "Spouse"...